src/wrapper/Websocket.ts

```diff
@@ -185,10 +185,15 @@ export default class Websocket {
                 const baseUrl = (
                     (await core.Supplier.get(this.options.environment)) ?? environments.CartesiaEnvironment.Production
                 ).replace(/^https?:\/\//, "");
-                const params = {
-                    api_key: await core.Supplier.get(this.options.apiKey),
-                    cartesia_version: this.options.cartesiaVersion,
+                const params: Record<string, string> = {
+                    cartesia_version: this.options.cartesiaVersion || "2024-06-10",
```

Comment:
why is this change needed? just curious

Reply:
to make the type of `cartesia_version` be `Record<string, string>`